### ISSUE TITLE
Spyder Issue when using ctrl+P : goto line 

### ISSUE THREAD

**Issue by Animadversio** [spyder-ide/spyder#4989]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "C:\Program Files\Anaconda3\lib\site-packages\spyder\widgets\fileswitcher.py", line 755, in setup
    self.setup_file_list(filter_text, current_path)
  File "C:\Program Files\Anaconda3\lib\site-packages\spyder\widgets\fileswitcher.py", line 629, in setup_file_list
    text_item += " [{0:} {1:}]".format(self.line_count[index],
AttributeError: 'FileSwitcher' object has no attribute 'line_count'

## Version and main components

* Spyder Version: 3.2.1 
* Python Version: 3.5.2
* Qt Versions: 5.6.0, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.3.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.18.1 (OK)
numpy >=1.7      :  1.12.1 (OK)
sphinx >=0.6.6   :  1.4.6 (OK)
rope >=0.9.4     :  0.9.4-1 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
nbconvert >=4.0  :  4.2.0 (OK)
sympy >=0.7.3    :  1.0 (OK)
cython >=0.21    :  0.24.1 (OK)
qtconsole >=4.2.0:  4.2.1 (OK)
IPython >=4.0    :  5.1.0 (OK)
pylint >=0.25    :  1.5.4 (OK)

```

**Comment by ccordoba12** [comment]
Thanks for reporting. We'll fix this in our 3.2.2 version.

**Comment by ccordoba12** [comment]
@dalthviz, please take a look at this one.
